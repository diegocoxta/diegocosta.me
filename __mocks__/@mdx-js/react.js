module.exports = {
  MDXProvider: jest.fn(({ children }) => children),
};
