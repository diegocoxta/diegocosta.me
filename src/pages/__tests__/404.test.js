import React from 'react';
import { render } from '@testing-library/react';

jest.mock('../../components/Header', () => () => <p>Header</p>);
jest.mock('../../components/Footer', () => () => <p>Footer</p>);
jest.mock('../../components/Metatags', () => () => <p>Metatags</p>);

import NotFoundPage from '../404';

describe('<NotFoundPage>', () => {
  it('should render properly', () => {
    const { baseElement } = render(<NotFoundPage />);
    expect(baseElement).toMatchSnapshot();
  });
});
