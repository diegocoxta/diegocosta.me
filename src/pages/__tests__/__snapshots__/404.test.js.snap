// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFoundPage> should render properly 1`] = `
.c0 {
  max-width: 800px;
  margin: 0 auto;
  padding: 0 20px;
}

.c1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  margin: 40px 0 0 0;
  -webkit-box-pack: end;
  -webkit-justify-content: flex-end;
  -ms-flex-pack: end;
  justify-content: flex-end;
}

.c2 {
  margin: 0 5px;
  text-transform: uppercase;
  color: #0C0D0E;
  font-weight: 700;
  -webkit-text-decoration: none;
  text-decoration: none;
}

.c3 {
  background: linear-gradient(90deg,#FFFFFF 20px,transparent 1%) center, linear-gradient(#FFFFFF 20px,transparent 1%) center, #0C0D0E;
  background-size: 22px 22px;
  height: 120px;
  margin-top: 40px;
}

.c4 {
  margin: 0 0 60px 0;
  font-weight: 400;
  color: #0C0D0E;
}

.c5 {
  margin: 20px 0 0 0;
}

.c6 {
  font-size: 36px;
  margin: 0;
}

.c7 {
  color: #737373;
  font-size: 18px;
  margin: 5px 0;
}

.c8 {
  line-height: 1.5;
}

.c8 a {
  color: #626D4A;
  -webkit-text-decoration: none;
  text-decoration: none;
  border-bottom: 1px solid #FFFFFF;
}

.c8 a:hover,
.c8 a:focus {
  border-bottom: 1px solid #626D4A;
  outline: none;
}

.c8 blockquote {
  border-left: 5px solid #626D4A;
  padding-left: 20px;
}

.c8 .gatsby-highlight {
  padding: 10px 0;
}

.c8 pre[class*='language-'],
.c8 code[class*='language-'],
.c8 .gatsby-highlight {
  background-color: #FFFFFF;
  font-family: monospace;
  white-space: pre-wrap;
}

.c8 .line-numbers-rows,
.c8 .line-numbers-rows span::before {
  border: 0.5px solid #FFFFFF;
}

.c8 img {
  box-shadow: none !important;
}

.c8 h3 {
  font-size: 28px;
}

<body>
  <div>
    <section
      class="c0"
    >
      <main>
        <div
          class="c1"
        >
          <a
            class="c2"
            language="pt"
          >
            pt
          </a>
        </div>
      </main>
    </section>
    <p>
      Header
    </p>
    <main>
      <p>
        Metatags
      </p>
      <div
        class="c3"
      />
      <section
        class="c0"
      >
        <main>
          <article
            class="c4"
            data-testid="article-item"
          >
            <header
              class="c5"
            >
              <h2
                class="c6"
              >
                404page.title
              </h2>
              <p
                class="c7"
              >
                 
              </p>
            </header>
            <section
              class="c8"
            >
              404page.message
            </section>
          </article>
        </main>
      </section>
    </main>
    <p>
      Footer
    </p>
  </div>
</body>
`;
